tox.ini

```diff
@@ -386,7 +386,7 @@ commands = pylint \
 skipsdist = True
 skip_install = True
 deps = tomte[safety]==0.2.14
-commands = safety check -i 37524 -i 38038 -i 37776 -i 38039 -i 39621 -i 40291 -i 39706 -i 41002 -i 51358 -i 51499
```

Comment:
Can you remind me what are we skipping here?

Reply:
https://data.safetycli.com/v/67599/97c/